app/models/item_analytics.rb

```diff
@@ -0,0 +1,9 @@
+module ItemAnalytics
+
+  def self.most_popular
+    item_quantities = Item.joins(:orders_items).group(:item_id).sum(:quantity)
+    max_quantity = item_quantities.values.max
+    Item.find(item_quantities.key(max_quantity))
```

Comment:
something like `Item.joins(:orders_items).group(:item_id).order('sum(quantity) DESC').first` might work so you don't have to drop down to ruby.